nodejs/week2/homework/meal-sharing/src/backend/app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const app = express();
+
+const mealsRouter = require("./api/meals-router");
+const reservationsRouter = require("./api/reservations-router");
+const reviewsRouter = require("./api/reviews-router");
+
+// app.use binds middleware to your application. You can give app.use a path and router. The mini router will take care of all requests with the path
+app.use("/api/meals", mealsRouter);
+app.use("/api/reservations", reservationsRouter);
+app.use("/api/reviews", reviewsRouter);
+
+module.exports = app;
```

Comment:
very nicely done and clean

Reply:
hi Stephen thanx once again for your valuable feedback. please have a look on the suggested changes commit where I have amended the code according to your suggestions. let me know if I could merge the pull request. thank you